docs/source/notebook.rst

```diff
@@ -175,14 +182,15 @@ cell**.
 **notebook name**: The name of the notebook document is displayed at the top
 of the page, next to the ``IP[y]: Notebook`` logo. This name reflects the name
 of the ``.ipynb`` notebook document file.  Clicking on the notebook name
-brings up a dialog which allows you to rename it. Thus, renaming a notebook
+brings up a dialog which allows you to rename the notebook, which also renames
+the file containing its code. Thus, renaming a notebook
```

Comment:
This wording is distracting to me. I think we're overemphasising that the notebook name is the file name (which would be my default assumption) because in very early versions of the notebook that wasn't true, and you could re-title the notebook without renaming the file. We changed that pretty quickly because it had some surprising behaviour.